packages/components/src/local/organisms/setting-platform-types/index.tsx

```diff
@@ -130,6 +130,21 @@ export const SettingPlatformTypes: React.FC<PropTypes> = ({ platformType, onChan
       }
     }
 
+    const handleSave = (): void => {
```

Comment:
thanks @lilitkarapetyan - the changes in this file are great.

Nice & compact, clearly named, good comments. 

Thanks :-)